jplag.frontend.csharp-6/src/main/java/de/jplag/csharp/CSharpListener.java

```diff
@@ -215,6 +215,7 @@ public void enterLockStatement(LockStatementContext context) {
 
     @Override
     public void enterUsingStatement(UsingStatementContext context) {
+        // TODO DF: This causes a null pointer.
         transformToken(USING, null);
```

Comment:
Good point. If I understand [using statements](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement) correctly, this should be `transformToken(USING, context.getStart());`